oauth-common/src/main/java/io/strimzi/kafka/oauth/common/Config.java

```diff
@@ -0,0 +1,135 @@
+package io.strimzi.kafka.oauth.common;
+
+import java.util.Properties;
+
+public class Config {
+
+    public static final String OAUTH_CLIENT_ID = "oauth.client.id";
```

Comment:
Is `OAUTH_CLIENT_ID` and `OAUTH_CLIENT_SECRET` ever used in server? If not, should it be in some ClientConfig class in the client?

Reply:
Yes, they are used. Every client connecting to Keycloak server needs to identify with a clientId. That includes client applications that perform authentication with the Keycloak server to get an access token, as well as server applications which accept an access token, and validate it with Keycloak server. So, this is a notion of Keycloak client or more generally of OAuth2 authorization server's client. As opposed to Kafka where Kafka broker is the server, and whatever connects to the Kafka broker is the client. For Keycloak these are both OAuth clients and need to present their clientId and secret.

Reply:
So, does this mean that this needs to be configured in the server as well? If yes, what should be the values? When working on the Strimzi integration I didn't configured this and it didn't seemed that anything complained. (I used the JSON you have in this repo which I imported)

Reply:
When JWTSignatureValidator is used (that is when OAUTH_JWKS_ENDPOINT_URI is configured) only public Keycloak endpoints are used which don't require client to authenticate ... it only fetches certs from server. But if OAuthIntrospectionValidator is used (that is when OAUTH_INTROSPECTION_ENDPOINT_URI is configured) the requests to it have to authenticate with client credentials (clientId + secret).

Reply:
Ok, I have to add it then. Thanks for the explanation.